test/e2e/node/pre_stop.go

```diff
@@ -219,6 +219,46 @@ var _ = SIGDescribe("PreStop", func() {
 			return false, err
 		})
 	})
+
+	/*
+	Usecase: PreStop hook is not executed for completed/terminated pod on issuing a DELETE API call
+	Description: SIGTERM and PreStop hook calls are made simultaneosly once a DELETE API call is made. However, if the main command finishes
+					execution before a DELETE API call then the pod enters the completed state. A DELETE API call would then kill and
+					remove the pod from the namespace without calling the preStop hook. Under this situation, the FailedPreStopHook
+					warnign event too would be reflecting false indicating no error.
+	Approach: 1. Sleep till the pod completes it main command and enters completed state
+			  2. Verify that the pod is alive
+			  3. Issue a DELETE API call
+			  4. Poll immediately to check if the pod is alive. If it is destroyed then preStop wasn't executed
+	*/
```

Comment:
nits:
- warnign -> warning
- Unnecessary tab spaces are contained at the beginning of lines

Also, try "go fmt" for formarting your code.

Reply:
Corrected the 'warning'. I am unable to understand what TAB spaces are being referred to here. Kindly guide